pkg/hintrunner/zero/zerohint_uint256.go

```diff
@@ -121,3 +127,71 @@ func createUint256AddHinter(resolver hintReferenceResolver, low bool) (hinter.Hi
 
 	return newUint256AddHint(a, b, carryLow, carryHigh, low), nil
 }
+
+func newSplit64Hint(a, low, high hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "Split64",
+		Op: func(vm *VM.VirtualMachine, _ *hinter.HintRunnerContext) error {
+			//> 	ids.low = ids.a & ((1<<64) - 1)
+			//> 	ids.high = ids.a >> 64
+
+			a, err := hinter.ResolveAsFelt(vm, a)
+			if err != nil {
+				return err
+			}
+			var aBig big.Int
+			a.BigInt(&aBig)
+
+			// Calculate low value
+			mask := new(big.Int).SetUint64(^uint64(0))
+			lowBig := new(big.Int).And(&aBig, mask)
+			low64 := lowBig.Uint64()
+			lowValue := memory.MemoryValueFromUint(low64)
+
+			lowAddr, err := low.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+
+			err = vm.Memory.WriteToAddress(&lowAddr, &lowValue)
+			if err != nil {
+				return err
+			}
+
+			// Calculate high value
+			highBig := new(big.Int).Rsh(&aBig, 64)
+			highValue := memory.MemoryValueFromFieldElement(new(fp.Element).SetBigInt(highBig))
+
+			highAddr, err := high.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+
+			err = vm.Memory.WriteToAddress(&highAddr, &highValue)
```

Comment:
Could do a `return vm.Memory.WriteToAddress(&highAddr, &highValue)`, since the error is returned as is.